manifests/ruby.yml

```diff
@@ -609,14 +609,22 @@ tests/:
     Test_Config_HttpServerErrorStatuses_FeatureFlagCustom: missing_feature
     Test_Config_IntegrationEnabled_False:
       '*': irrelevant (endpoint not implemented)
-      rails70: v2.0.0
+      rails70: v2.12.0
     Test_Config_IntegrationEnabled_True:
       '*': irrelevant (endpoint not implemented)
-      rails70: v2.0.0
-    Test_Config_LogInjection_128Bit_TraceId_Disabled: incomplete_test_app (endpoint not implemented)
-    Test_Config_LogInjection_128Bit_TraceId_Enabled: missing_feature (not enabled by default)
-    Test_Config_LogInjection_Default: incomplete_test_app (endpoint not implemented)
-    Test_Config_LogInjection_Enabled: incomplete_test_app (endpoint not implemented)
+      rails70: v2.12.0
+    Test_Config_LogInjection_128Bit_TraceId_Disabled:
+      '*': irrelevant (endpoint not implemented)
+      rails70: v2.12.0
+    Test_Config_LogInjection_128Bit_TraceId_Enabled:
+      '*': irrelevant (endpoint not implemented)
+      rails70: v2.12.0
+    Test_Config_LogInjection_Default:
+      '*': irrelevant (endpoint not implemented)
+      rails70: irrelavent (dd-trace-rb enables log injection by default, we are planning to revist this behavior in the future. This will likely be the new default)
+    Test_Config_LogInjection_Enabled:
+      '*': irrelevant (endpoint not implemented)
+      rails70: v2.12.0
```

Comment:
```suggestion
      rails70: v2.13.0
    Test_Config_LogInjection_128Bit_TraceId_Disabled:
      '*': irrelevant (endpoint not implemented)
      rails70: v2.13.0
    Test_Config_LogInjection_128Bit_TraceId_Enabled:
      '*': irrelevant (endpoint not implemented)
      rails70: v2.13.0
    Test_Config_LogInjection_Default:
      '*': irrelevant (endpoint not implemented)
      rails70: irrelavent (dd-trace-rb enables log injection by default, we are planning to revist this behavior in the future. This will likely be the new default)
    Test_Config_LogInjection_Enabled:
      '*': irrelevant (endpoint not implemented)
      rails70: v2.13.0
```